webpack/assets/javascripts/bundle_novnc.js

```diff
@@ -48,24 +49,32 @@ function connectFinished() {
   showStatus('normal', __('Connected'));
 }
 
+function on_error(e) {
+  if (e.code == 1006) {
+      showStatus('failed', __('The connection was closed by the browser. please verify that the certificate authority is valid'));
```

Comment:
You have to import the `__` method, see: https://github.com/theforeman/foreman/blob/4e3c749bea288a05b4d336723563c83d650ff681/webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js#L6